news/2017-10-16-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,72 @@
+
+**For osu!catch:** [Electoz](https://osu.ppy.sh/users/6485263) and [alienflybot](https://osu.ppy.sh/users/636114)
+
+**For osu!mania:** [Asherz007](https://osu.ppy.sh/users/9014047), [Litharrale](https://osu.ppy.sh/users/3189995) and [Weber](https://osu.ppy.sh/users/6410432)
+
+As mentioned previously, we replaced the Tier system with the probation phase, which lasts for two months at most. In this time the Nominator is able to bubble beatmaps, but not to qualify them. You could call it a “test run”, where the probationary BNs are closely supervised by the QAT. Having joined as probationary Beatmap Nominators several weeks ago, these new members have demonstrated an acceptable level of activity and quality in this time. As such, they have recently become full members of the Beatmap Nominators who can both bubble and qualify beatmaps.
+
+### The Qualification Cap
+
+osu! continues to grow every day, and the mapping community is no exception. To account for the huge amount of new beatmaps aiming to reach Ranked status, **Beatmap Nominators can now qualify 3 beatmaps per day, up from just 1.** This, along with the removal of the Beatmap Nominator Tier system, meant that the amount of Qualified beatmaps literally doubled in the space of just one month, if not more! 
+
+Because of these factors, beatmaps may have spent up to 12 days or even more in the Qualified stage. Since this was awfully slow and unproductive, **we raised the amount of beatmaps that can leave the Qualified stage per day to 8, up from 6.** This will shorten the time that a beatmap set will remain in qualified, hopefully back down towards the target of 7 days.
+
+## Within the Quality Assurance Team
+
+Unfortunately, after nearly three years of service, our resident metadata expert [IamKwaN](https://osu.ppy.sh/users/1856463) has decided to step down from the QAT in order to seek some well-earned rest and relaxation. Her monumental efforts in the fields of metadata checking and standardisation have truly been a credit to osu! and the mapping community, and her absence will be a great loss to our team. Additionally, one of our newest team members, [Cryptic](https://osu.ppy.sh/users/3728904) has also departed after a short tenure to focus on IRL duties. You will be missed guys, thanks for your contributions!
+
+It’s not all sad news however, as we also have **two new additions too!** Please give a warm welcome to the Quality Assurance Team for [Naxess](https://osu.ppy.sh/users/8129817) and [Raiden](https://osu.ppy.sh/users/2239480)! Together with Mao, Naxess will assist with development of the Beatmap Nominator subdivisions, where his experience coding a Discord bot for logging Beatmap Nominator activity will be invaluable. Raiden has been responsible for a large amount of work on the new osu!taiko Ranking Criteria, and has joined to help manage the osu!taiko Beatmap Nominators who have grown a lot with the last phase of applications.
+
+## Mapping with Rewards is back!
+
```

Comment:
Newline at All you have to do

Change to: All you have to do is map one of the listed tracks, upload them to the BSS and then submit them to the contest page, and you're done!

Reply:
Reworded this and added all above newlines